core-common/src/main/java/org/uniprot/core/util/Utils.java

```diff
@@ -141,4 +141,10 @@ public static boolean notNull(@Nullable Object o) {
         if (nullOrEmpty(str)) return str;
         return Character.toLowerCase(str.charAt(0)) + str.substring(1);
     }
+
+    public static void putIfNameAndValueNotNull(String key, Object value, Map<String, Object> map) {
```

Comment:
We should add test cases for this in `Utils`, so that others know what the desired behaviour is. looking at the implementation, then:
1 for addition when successfully added
1 when value is null
1 when key is null
1 when key is empty
1 value is empty
1 when the same key is added 2 times

is that all?